### PR TITLE
[new downloader] Added countries names localization

### PR DESCRIPTION
**Pull request by kshalnev**

### CODE REVIEW

File: storage/country_name_getter.hpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  /// \brief Sets a locale.
+  /// @param locale is a string representation of locale. For example "en", "ru", "zh-Hant" and so on.
+  /// \note See countries/languages.txt for the full list of available locales.
+  void SetLocale(string const & locale);
+
+  string operator()(TCountryId const & countryId) const;
+
+  // for testing
+  void SetLocaltForTesting(string const & jsonBuffer, string const & locale);
```
Comment (trashkalmar) [inline storage/country_name_getter.hpp]: Locale

Comment (kshalnev) [inline storage/country_name_getter.hpp]: поправил

**Comment by trashkalmar** [comment]
LGTM

**Comment by bykoianko** [comment]
LGTM

**Pull request merged**